bind.go

```diff
@@ -113,7 +113,7 @@ func In(query string, args ...interface{}) (string, []interface{}, error) {
 		v := reflect.ValueOf(arg)
 		t := reflectx.Deref(v.Type())
 
-		if t.Kind() == reflect.Slice {
+		if t.Kind() == reflect.Slice && t != reflect.TypeOf([]byte{}) {
```

Comment:
Please add a comment explaining this exception and add a test case to TestIn in sqlx_test.go.